app/javascript/components/table/Row.vue

```diff
@@ -79,11 +84,15 @@
 
     computed: {
       hasMetadata () {
-        return this.item.metadata.length !== 0
+        return this.item.metadata && this.item.metadata.length !== 0
       },
 
       hasFactsheet () {
-        return this.item.factsheet.length !== 0
+        return this.item.factsheet && this.item.factsheet.length !== 0
+      },
+
+      hasResourceLink () {
+        return this.item.website_download_link && this.item.website_download_link !== 0
```

Comment:
Similar thing here for the string. You could use `!!this.item.website_download_link`
`!` will transform the string into a boolean depending on its presence.
`!''` is true, while `!'string'` is false because the string exists and negating it with `!` returns false.
So adding an extra `!` (so `!!`) will do the trick

Reply:
![Big fan of it!](https://img00.deviantart.net/6aec/i/2006/033/7/0/vivi_ornitier_by_jisuk.jpg)